### ISSUE TITLE
[BUG]: Fatal Exception: android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord

### ISSUE THREAD

**Issue by mezalejandro** [TheWidlarzGroup/react-native-video#3897]
### Version

^6.1.2

### What platforms are you having the problem on?

Android

### System Version

11

### On what device are you experiencing the issue?

Real device

### Architecture

Old architecture

### What happened?

On firebase, I'm getting a lot of error alerts:
<img width="555" alt="Screenshot 2024-06-11 at 11 32 33 AM" src="https://github.com/TheWidlarzGroup/react-native-video/assets/568141/fca03eb3-36a8-49d7-a6f1-51680dd57e07">

### Reproduction

repository link

### Reproduction

Step to reproduce this bug are:

**Comment by freeboub** [comment]
do you use showNotificationControls ?

**Comment by mezalejandro** [comment]
Hello @freeboub no, i don't use it.
I disabled showNotificationControls but the mistake it still.
Thanks

**Comment by freeboub** [comment]
OK, but are you sure this is related to react native video ?
I don't see any proof or clue to point react native video package ...

**Comment by mezalejandro** [comment]
Hello @freeboub, I copied the issue from firebase:
    
``` 
Fatal Exception: android.app.ForegroundServiceDidNotStartInTimeException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{b7c2b81 u0 com.myapp/com.brentvatne.exoplayer.VideoPlaybackService}
       at android.app.ActivityThread.throwRemoteServiceException(ActivityThread.java:2147)
       at android.app.ActivityThread.access$2800(ActivityThread.java:310)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2376)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loopOnce(Looper.java:226)
       at android.os.Looper.loop(Looper.java:313)
       at android.app.ActivityThread.main(ActivityThread.java:8663)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
```

**Comment by freeboub** [comment]
Ok, clear, I don't understand because the service shall started only if showNotificationControls is true 🤔

**Comment by ashwanihundwani1986** [comment]
@mezalejandro Can you help with this please if you found any solutions to this?

**Comment by mezalejandro** [comment]
Hello @ashwanihundwani1986, The error still appears in firebase, I have many errors about this

**Comment by freeboub** [comment]
I think this issue is an exoplayer issue see: https://github.com/androidx/media/issues/167

**Comment by ashwanihundwani1986** [comment]
@freeboub There are some solutions being suggested on this thread, please check those
Few things are to update to gradle 8.4 and checking migration does for exoplayer3.

**Comment by freeboub** [comment]
> @freeboub There are some solutions being suggested on this thread, please check those
> Few things are to update to gradle 8.4 and checking migration does for exoplayer3.
> 

No time on my side for now..  but if you find a fix a PR would be welcome!

**Comment by freeboub** [comment]
linked PR has been merged in 6.6.4, I will close the issue, let me know if this is reproduced with this version.
